packages/nhsuk-frontend/src/nhsuk/components/skip-link/_index.scss

```diff
@@ -16,6 +17,15 @@
   top: nhsuk-spacing(3);
   padding: nhsuk-spacing(2);
   @include nhsuk-visually-hidden-focusable; // [1]
+
+  // Respect 'display cutout' safe area (avoids notches and rounded corners)
+  @supports (padding: string.unquote("max(calc(0px))")) {
```

Comment:
Love this PR @paulrobertlloyd

I suppose for completeness this `@supports` test should be for `left:` too?

Reply:
Fixed